virttest/utils_net.py

```diff
@@ -3895,16 +3895,21 @@ def get_default_gateway(iface_name=False, session=None):
     :return: A string of the host's or guest's default gateway or interface.
     :rtype: string
     """
+    if ip_ver == 'ipv4':
+        ip_cmd = 'ip'
+    elif ip_ver == 'ipv6':
+        ip_cmd = 'ip -6'
```

Comment:
I wonder if it's better to add an 'else' here to avoid 'undefine' error when 'ip_ver' is neither 'ipv4' nor 'ipv6'. what do you think?

Reply:
I guess, which type of exception do you prefer?

Reply:
I think it's ok to set a default value or raise an exception. or something like below:
```suggestion
    if ip_ver == 'ipv6':
        ip_cmd = 'ip -6'
    else:
        ip_cmd = 'ip'
```
its up to you.